packages/platform-android/package.json

```diff
@@ -5,15 +5,22 @@
   "main": "build/index.js",
   "dependencies": {
     "@react-native-community/cli-tools": "^2.7.0",
+    "@types/node": "^12.6.4",
     "chalk": "^2.4.2",
     "execa": "^1.0.0",
     "jetifier": "^1.6.2",
     "logkitty": "^0.5.0",
-    "slash": "^2.0.0",
+    "slash": "^3.0.0",
     "xmldoc": "^0.4.0"
   },
   "files": [
     "build",
     "native_modules.gradle"
-  ]
+  ],
+  "devDependencies": {
+    "@types/execa": "^0.9.0",
+    "@types/fs-extra": "^8.0.0",
+    "@types/glob": "^7.1.1",
+    "@types/xmldoc": "^1.1.4"
+  }
```

Comment:
we need "references" to "@react-native-community/cli-tools" now, see: https://github.com/facebook/jest/blob/481334b645348fc7471baa8b1a41d99c8e1d6017/packages/jest-changed-files/tsconfig.json#L7-L9